src/repositories/statuses.jl

```diff
@@ -23,10 +23,30 @@ Status(id::Real) = Status(Dict("id" => id))
 
 namefield(status::Status) = status.id
 
+type Webhook <: GitHubType
+    id::Nullable{Int}
+    url::Nullable{HttpCommon.URI}
+    test_url::Nullable{HttpCommon.URI}
+    ping_url::Nullable{HttpCommon.URI}
+    name::Nullable{GitHubString}
+    events::Nullable{Array{GitHubString}}
+    active::Nullable{Bool}
+    config::Nullable{Dict{GitHubString, GitHubString}}
+    updated_at::Nullable{Dates.DateTime}
+    created_at::Nullable{Dates.DateTime}
+end
+
+Webhook(data::Dict) = json2github(Webhook, data)
+
```

Comment:
Can you add this method (since the `id` field is the URL argument GitHub's API uses to specify webhooks):

```jl
namefield(hook::Webhook) = hook.id
```